packages/bridge-controller/CHANGELOG.md

```diff
@@ -7,6 +7,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- **BREAKING:** Bump dependency bignumber.js to ^9.1.2 ([#5614](https://github.com/MetaMask/core/pull/5614))
```

Comment:
Nit: this can be removed from the changelog
```suggestion
- **BREAKING:** Bump dependency bignumber.js to ^9.1.2 ([#5614](https://github.com/MetaMask/core/pull/5614))